### ISSUE TITLE
index.html issue with Markbot

### ISSUE THREAD

**Issue by riannazandbelt** [riannazandbelt/accessibility-fixer-upper#2]
Hi @HannahDelaney, 

I'm currently stuck with this assignment as I've failed 2 accessibility tests when I run the assignment through Markbot and I've tried to make changes but nothing seems to be working. I've attached a screenshot with the Markbot issue, to hopefully help you make sense of the problem. :)

Any help/guidance would be greatly appreciated!
Rianna

<img width="502" alt="Screen Shot 2020-04-06 at 2 59 46 PM" src="https://user-images.githubusercontent.com/55094049/78595022-4676e780-7817-11ea-8fa7-f5092affe539.png">

**Comment by HannahDelaney** [comment]
There is some quick rearranging of things that should solve these problems. The first "about" section is outside of the main navigation, so the skip link tagging is a bit messy. If you move your `<main>` tag to be above that first "about" section, it should be fine.

The second problem is because you don't have an `id` on the section about "contact" that is listed in the skip links. If you give that section an `id` that matches the skip links (ie. `id="contact"`) then you should be good!

Rearranging the main tag will likely cause a slight screenshot matching issue, but I'll let you sort those smaller details out! ;)

**Comment by riannazandbelt** [comment]
AHHH thank you! It worked :)
